ReadMe.Md

```diff
@@ -48,16 +48,16 @@ The number of tests run will be determined by the values provided in this file.
 
 In Postman at the collection level, click on **Run Collection**
 
-<span><img src="https://raw.githubusercontent.com/ConsumerDataStandardsAustralia/dsb-schema-tools/main/postman/images/Pm_Screen1.JPG?token=AETEGILKIOYBMKKWQT2M253BBBXHW" width="300" >
```

Comment:
I think it should be _main/images_ rather than _dev/images_. Any branch other than main (master) shouldn't be referenced.